whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/NrtmConnectionPerIpLimitHandler.java

```diff
@@ -23,29 +23,31 @@
 @ChannelHandler.Sharable
 public class NrtmConnectionPerIpLimitHandler extends SimpleChannelUpstreamHandler {
```

Comment:
Can you move the ACL logic into a separate class? Look at the #596 PR for an example (there's a separate class for AccessControlList).  It's confusing that this Handler now also includes the ACL logic, it should be separate.

Reply:
Earlier I thought of moving it to a different class but this class name is connection limit perIp handler and acl also works on per Ip limit so I put the logic in same class, but I not a big deal to put it in a seperate class if it makes it more readable

Reply:
Great, thanks!